packages/mobile/src/screens/sign-on-screen/components/layout.tsx

```diff
@@ -82,30 +86,32 @@ export const PageFooter = (props: PageFooterProps) => {
     >
       {/* Prefixes float above the shadowed paper container  */}
       {prefix ? <Flex ph={gutterSize}>{prefix}</Flex> : null}
-      <Paper
-        p='l'
-        justifyContent='center'
-        gap='l'
-        alignItems='center'
-        direction='column'
-        shadow='midInverted'
-        style={css({
-          borderRadius: 0,
-          paddingBottom: insets.bottom
-        })}
-        {...other}
-      >
-        <Button
-          fullWidth
-          disabled={!dirty || !isValid}
-          onPress={() => handleSubmit?.()}
-          {...buttonProps}
+      <KeyboardAvoidContainer style={{}} keyboardShowingOffset={spacing.unit5}>
```

Comment:
Funny you have to put empty? Can be optional?

Reply:
oop yeah I meant to update that but forgot. thanks!